azure/src/index.css

```diff
@@ -1,6 +1,7 @@
 *{
   box-sizing: border-box;
   margin: 0;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
```

Comment:
Let us create a separate CSS file for each components. A single CSS file will be difficult to manage as the application grows.